components/elements/Logo/LogoSVG.tsx

```diff
@@ -0,0 +1,35 @@
1q8FD+gBLPTIheR+KjOxcMRVD5kl6roL63mX8v9/x5d/conzJzEAyxVsDgttmJQS28PO9q+bstiqRZjMF2RsZDlWAX4bNLhZwA+wJJkvBI3zfuBDlUusWN9nfHknBseWwHZnplMwZGHzv9nUGtBI7+8CPenLrRhCpdWQx8mZTsyousAhXpAFRPD3pbha5gEsf+Ex5BPsWZh9Yt2O8diI8G9sFT6s4LOf+/qOKzi+CvBaHXV+0j/jMUS26CkEi7SREc/nG7NwzhrC1S2sd1EsxuEbwYOYiAVrWopcBwpm+NqwTXts/xiKWCsB6/pGM77g+Cjgzjrr/b2vt+JOTiuh/0I2bphn8hQJ84Atm1zXCr7Hezl4AH2+R1oac6IPTZ4uoc65YUH9m2DD7QOleihMHQNBtDFM4V5X8m7/JcvS3Z7SLj0k8NHgeceNroslj8mCQL1EE4LNY5Yso8l3ULLGfyk+uSHmIpqpcmYX67nqlCFLKjmXMsMR8AcvwzeCY+fTmBjga5OPBrc141kPQKZ1g3dSNG1IqwXalnxV9Tx1OmhjernNMcX6I/RnBubAtG5w/p5BQ32VJvTQwLm+/LLxEoFf+/NODY49SIMSv2NK+4d9HTMxk62SOWyaDabQz2iPkNqYkjlrkLOxODKrVr7yfUvrbTHbwRso7rn2hP+80Ltub3JPxUcb9dIL6tgDmwveTwWHevJ8d5cG9/Zgg+UZAvwMm6KA7QLdgQUXXb2RdVUhy9bBOzqu8hUtAksRMS4Qbp5/gaOxVe9R2JzvdH9sDJYAs1ismqxx/YwSe59Y75ktoO6l2lQNtd3TplgOu0lUF502Wzyd6//fC/hpo+XyZa/qG/8U+vMUtr25M1X65dQhwzeCeiunVWslmCL0GPobIFTDbGwf+dfY9lrZXhXb2svmqv+mUUGE+texJuYpNxfYvsprHvMyneD/H02TTfD9Mz8YcyAL59VgW3X3YoueXRiArrNC3X/z9cwjUkaGakyrFpH0CUlbSRohMwgdKjPunC0z+HzZ/zwv6blawm0AoyUdIjMY/Zhz7qWa7qBy+WtKGuPlPsw5N7qKa4Yrt4De1V9/q3OucnraBuEb2+6SPidpN5nhbMg8SY9L+qek+yXd55wbkA4U2ETSI5IW8eWUtf3sSrDhKVNoH1D5iprL3wAzLgU4qobrjgt6+SWx+ew3K1/ZHDC7xS29XHfSPyhByMuYd+SR2LSk4uaAfwfPBWXs1Yp7ajswRyAwNUdDww1jltvvYPPYo2u4bmnyUB6X+WN/aPSwWA9+ON8Jm4ffR77wK6QP2yy4AjME2Ruzkt8C2wr9KXnkYoArYt9bNLCwGwAPNLBMhxk7zPE/NfW4wG+8TO9h25HrAL9olHzNANtP/xRwKrbQDE3rquV6Grix0HFg8Q/Boj8MKBZMQXnDyfPWvU2NJnG+18g0Auf4Y6fRYjVLvWB7/bthw/rV2G7WnCINcAGm/z2QNjRtaynYfCxTZwxYbYLtcR8VlPUAA1Agk1uNv+1lGwL8eqBytROY1dCGWODSnbF99pos4rsecv3WPGqMhIoFhD+UfB+5D1NH1WwCBWwV9BhH+2PbAPvUWlaig8F0eGA7ED+rNGRj+6gnk0fZmo8pqQccro3chGo+Xs+GWQ81fCco0cZgqoszg/nam9ge8j7YPG4bzOX0fD/Hyc57D1vpjmyADHf5Mh8PjlWdNTXRZWDGpw9Qmecxl4Oq9syrrHu8L/vm4FhHxh/qVNoq+r5z7h5Jo3xPt7OkTWUhOt6T9LqkhyXd45wbULSwCmST+VnBsdlNqCdRgrZqjBk+6PtTFU9sLDMkLa3+225zWyxDT9MWVsZtwlv+d+joNQ5YOYYwvUhqjDkP+98fJHfS/5ekxmWASpQlNcacu4K/D5Qk59wMmSVLItE6sN2WLMzcG3jDCK9WSl/aRGuhv+/wGf7YYBqZrDGRqAbMNCtzn30PMzoVbRSXO9FD0N+ZfQxmkjYoraoTUQD+N2iQB/hji9MFKXoTHQZmapXZ/4WLmbSQSbQezHAj4/ux5Un0MMBKWKQHsFiRqVdMxIP+eQjjBl/vAdK3vTy3BX+3JpljD5MaY3kmBH8vH02KHiE1xvKEkTFKpTtLNIjUGMsTxtb5TzQpeoTUGMuzWfD3c9GkSCTI/XEWECkyVyKRxdzJwtK1PPdyL5KG6dLsKynz327LvH+JHgELPZexVWx5Ej0KsDoWWQICp/5Ec0nDdHGOUv5sLowpSKKH8QuXqb5XfJMmB3ZP5KSecWGOkDnzS9K5zrlZ5U5OJJoCFmTzTd8rTqVEzulEoukAPw5W0D+OLU+iR8GSa07zDfE/QDKMSMQBOC/oFQ+OLU+iR8FS/mYZAh5JLgaJKGB+0VnU2vnAqNgyJXoU4JBgeD43tjyJHgVLopmlIX6FNsqClegxyBMYLQB2ii1PokcBDgiG53Niy5PoUbBkRm8Eq+fezZmXiAv9I41tH1ueRA+DZRAFeIFeT+DYRvSqcnd9//sfzjmiSpJ4n15tjFlmrX9HlSLRj15tjFmyob6oUiT60auN8R3/O62i24hebYxZKriUxaCN6NXGONb/3iaqFIkEsFGwDTgitjyJHgd4xjfII2LLkjB6dZiWpNH+995RpUgkvPlYnzeoXTe2PIkeBzjbD9Xnx5Yl0eNg6TX6/E+K2R2ZXp4zyjn3lqTfSBoq6cjI4iR6HWA573owHVgptjyJHidwykpzx0RcvLvq3VjyyrSyTsTF78rMAm6NLUsiIeBoP1zvF1uWRI8DDMJieb8BLF35ikSiiQBrAFOAC2LLkkgI+BQwF9g2tiyJhIATgOdIMb0TsfHqnuuBM2LLkkhkMb4fA3aMLUsiIWBN3yCXiy1LIiFgV+Cq2HIkEpIk4HDg8NhyJBKSJOA4YIvYciQSkiTg22l3JtEW+C3Dg1L0skRb4IONpmgUifYAGJp2ZxrL/wNOVeRg2DK4hQAAAABJRU5ErkJggg==' />
```

Comment:
In fact, this is a PNG icon, not an SVG. Try to remove the svg from the layout, because there is too much unnecessary code

Reply:
change svg image to png